.github/workflows/release-pipeline.yml

```diff
@@ -117,32 +117,42 @@ jobs:
     name: Generating Changelog
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Needed for changelog generation
 
       - name: Create new branch
-        run: git checkout -b actions/changelog-$(cat VERSION)
+        run: git checkout -b actions/changelog
+
+      - name: Set branch upstream
+        run: git push -u origin actions/changelog
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Setup python
         uses: actions/setup-python@v2
         with:
           python-version: '3.8'
 
+      - name: Install dependencies
+        run: pip install -e .[all]
+
       - name: Generate newest changelog
         run: gitchangelog $(cat VERSION) > CHANGELOG.md
 
       - name: Make commit for auto-generated changelog
-        uses: EndBug/add-and-commit@v4
+        uses: EndBug/add-and-commit@v7
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           add: 'CHANGELOG.md'
+          branch: actions/changelog
           message: '!gitchangelog'
 
       - name: Create pull request for the auto generated changelog
         run: |
           echo "PR_URL=$(gh pr create \
             --title "Auto-generated changelog for release $(cat VERSION)" \
             --body "This is an auto-generated PR" \
-            --reviewer microsoft/quilla-admin \
```

Comment:
the cli doesn't recognize the team name to add as a reviewer- I'll check to see if that's just a problem with the team or if that's a problem with the CLI